<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWidget</class>
 <widget class="QWidget" name="OptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="difficultyGroup">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>130</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radEasy">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>2 - 5 Letter Words</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>EASY</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radMedium">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>6 - 10 Letter Words</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>NORMAL</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radHard">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>10+ Letter Words</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>HARD</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>SELECT DIFFICULTY</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_edit_words">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>340</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit Words</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtCancel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>89</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtSave">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>89</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
